portality/upgrade.py

```diff
@@ -55,10 +55,7 @@ def do_upgrade(definition, verbose):
             tasks = tdef.get("tasks", None)
             if tasks:
                 for func_call, kwargs in tasks.iteritems():
-                    if args:
```

Comment:
your change seems sensible, but it kind of looks like this should have been failing with "name args is not defined" - or else it did get used for something